demo/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Cliquetis demo</title>
+  <style>
+  .log {
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 50%;
+    height: 100%;
+    padding: .4em;
+    font-size: .8em;
+    color: white;
+  }
+  </style>
+</head>
+<body>
+  <h1>Todo list</h1>
+  <p>Last modified: <span id="lastModified"></span></p>
+  <ul id="todo"></ul>
+  <form>
+    <p>
+      <input type="text" name="label" placeholder="Thing">
+      <input type="submit" value="Add">
+    </p>
+  </form>
+  <p>
+    <input type="button" id="clearCompletedBtn" value="Clear completed">
+    <input type="reset" id="clearAllBtn" value="Clear collection">
+    <input type="button" id="syncBtn" value="Synchronize collection">
+  </p>
+  <p>
+    <textarea id="log" class="log"></textarea>
+  </p>
+  <script src="../dist/cliquetis.js"></script>
+  <script>
+  function main() {
+    const db = new Cliquetis({remote: "http://0.0.0.0:8888/v0"});
```

Comment:
You are right. 
First I would have thought that the number comes from the version of the protocol (for example this PR is only compatible with ``kinto/v0``). But since the version number is the one of the Cliquet project, we cannot set it here as constant.

The only thing we have to have to get it from server is to read the location header of the ``/`` response